backend/battling/views.py

```diff
@@ -51,25 +49,56 @@ def form_valid(self, form):
         else:
             messages.success(self.request, "Battle ended! Check e-mail for results.")
 
-            # run_battle_and_send_email(battle)
+            get_battle_winner(battle)
 
         return super().form_valid(form)
 
 
-class DeleteTeam(DeleteView):
-    template_name = "battling/delete_team.html"
+class DeleteBattle(DeleteView):
+    template_name = "battling/delete_battle.html"
     success_url = reverse_lazy("home")
-    queryset = Team.objects.all()
+    queryset = Battle.objects.all()
```

Comment:
Here there's no filter for current user. Are we tackling this later?

Reply:
Not really, because either the creator or the opponent can delete the battle

Reply:
But in this case any user from the system can delete battles from others?